components/clarinet-sdk/src/index.ts

```diff
@@ -222,6 +225,15 @@ const getSessionProxy = () => ({
       return getDataVar;
     }
 
+    if (prop === "getBlockTime") {
+      const getBlockTime: GetBlockTime = () => {
+        const response = session.getBlockTime();
```

Comment:
⚠️

Reply:
What's the warning for here?

Reply:
Ah ok, that's what you were talking about on Discord. Let's follow up there